src/spork.cpp

```diff
@@ -389,6 +392,7 @@ bool CSporkMessage::CheckSignature(const CKeyID& pubKeyId, bool fSporkSixActive)
 {
     std::string strError = "";
 
+    // fSporkSixActive will only be true on testnet, it will be false on all other networks
```

Comment:
Maybe smth like to enforce this statement?
```suggestion
    // fSporkSixActive will only be true on testnet, it will be false on all other networks
    assert(fSporkSixActive == (Params().NetworkIDString() == CBaseChainParams::TESTNET));
```